patches/tModLoader/Terraria/ModLoader/UI/ModBrowser/UIBrowserFilterToggle.cs

```diff
@@ -14,6 +14,7 @@ internal class UIBrowserFilterToggle<T> : UICycleImage where T : struct, Enum
 		private set;
 	}
 
+	//TODO: Needs to update texture and logic
```

Comment:
So that it doesn't get lost as an outstanding TODO.
Will effect if add relevance mode.

Reply:
If added soon just let me know what you want, I sketched some of the texture for loading etc... so I can put down and change logic etc... just list me the possible states